.github/workflows/install-script.yml

```diff
@@ -44,6 +45,18 @@ jobs:
           mkdir -p "${CARGO_HOME}/bin"
           echo "CARGO_HOME=$CARGO_HOME" >> "$GITHUB_ENV"
 
+      - name: Set `BINSTALL_VERSION`
+        if: matrix.set_binstall_version != 'no'
+        env:
+          STRIP_V: ${{ matrix.set_cargo_home == 'with-v' && '1' || '' }}
```

Comment:
This is not working as I intended it to.

The variable is empty regardless of the value of `matrix.set_cargo_home`. I'll have to do a bit of research.

Reply:
I've pushed an update; instead of fighting with Github Actions expression syntax, the matrix value is copied directly into the environment variable and the step then tests for `without-v` in bash to make the strip-or-not call.

Reply:
Thank you, it has been merged!